zuul.d/projects.yaml

```diff
@@ -12,34 +12,40 @@
       - cifmw-kuttl
       - cifmw-edpm-build-images
       - cifmw-content-provider-build-images
-      - cifmw-molecule-polarion
-      - cifmw-molecule-os_must_gather
+      - cifmw-molecule-artifacts
+      - cifmw-molecule-build_openstack_packages
+      - cifmw-molecule-ci_setup
       - cifmw-molecule-copy_container
-      - cifmw-molecule-set_openstack_containers
-      - cifmw-molecule-os_net_setup
-      - cifmw-molecule-registry_deploy
-      - cifmw-molecule-openshift_provisioner_node
+      - cifmw-molecule-discover_latest_image
       - cifmw-molecule-edpm_build_images
-      - cifmw-molecule-pkg_build
+      - cifmw-molecule-edpm_deploy
+      - cifmw-molecule-edpm_deploy_baremetal
       - cifmw-molecule-edpm_prepare
-      - cifmw-molecule-repo_setup
-      - cifmw-molecule-tempest
+      - cifmw-molecule-hive
+      - cifmw-molecule-install_ca
+      - cifmw-molecule-install_yamls
+      - cifmw-molecule-libvirt_manager
       - cifmw-molecule-local_env_vm
       - cifmw-molecule-openshift_login
-      - cifmw-molecule-libvirt_manager
+      - cifmw-molecule-openshift_provisioner_node
       - cifmw-molecule-openshift_setup
-      - cifmw-molecule-artifacts
-      - cifmw-molecule-run_hook
-      - cifmw-molecule-rhol_crc
-      - cifmw-molecule-install_ca
-      - cifmw-molecule-install_yamls
       - cifmw-molecule-operator_build
-      - cifmw-molecule-discover_latest_image
-      - cifmw-molecule-test_deps
-      - cifmw-molecule-ci_setup
-      - cifmw-molecule-edpm_deploy
-      - cifmw-molecule-hive
-      - cifmw-molecule-build_openstack_packages
       - cifmw-molecule-operator_deploy
-      - cifmw-molecule-edpm_deploy_baremetal
+      - cifmw-molecule-os_must_gather
+      - cifmw-molecule-os_net_setup
+      - cifmw-molecule-pkg_build
+      - cifmw-molecule-polarion
+      - cifmw-molecule-registry_deploy
+      - cifmw-molecule-repo_setup
+      - cifmw-molecule-rhol_crc
+      - cifmw-molecule-run_hook
+      - cifmw-molecule-set_openstack_containers
+      - cifmw-molecule-tempest
+      - cifmw-molecule-test_deps
+      - cifmw-molecule-build_containers
```

Comment:
Can we drop this job from here? this pr https://github.com/openstack-k8s-operators/ci-framework/pull/285 includes this job with fixes. It will help to merge this pr.

Reply:
I think this is done. Here's the diff. Please let me know if I misunderstood. 

https://github.com/openstack-k8s-operators/ci-framework/compare/d85be4e1acbf64f76f44f511316cc12ce577e48c..46f76cd4b59cacd395c2c21d1ede4b25ea9951a4